db/migrate/20180124190321_create_customers.rb

```diff
@@ -1,13 +1,16 @@
 class CreateCustomers < ActiveRecord::Migration[5.1]
   def change
     create_table :customers do |t|
-      t.string :first_name
-      t.string :last_name
-      t.date :dob
-      t.string :address
-      t.string :postcode
-      t.string :city
-      t.string :country
```

Comment:
Can you set account_id at the beginning for the table

dob means (date of birth) can you move it back below last_name.